docs/content/preview/yugabyte-platform/create-deployments/async-replication-platform.md

```diff
@@ -18,101 +18,165 @@ Within the concept of replication, universes are divided into the following cate
-
 - A target universe is the recipient of the replicated data. One source universe can replicate to one or more target universes.
 
 For additional information on xCluster replication in YugabyteDB, see the following:
 
 - [xCluster replication: overview and architecture](/preview/architecture/docdb-replication/async-replication/)
 - [xCluster replication between universes in YugabyteDB](/preview/deploy/multi-dc/async-replication/)
 
-You can use the YugabyteDB Anywhere UI to set up and configure xCluster replication for universes whose tables do not contain data. In addition, you can perform monitoring by accessing the information about the replication lag and enabling alerts on excessive lag.
+You can use the YugabyteDB Anywhere UI to set up and configure xCluster replication for universes. In addition, you can perform monitoring by accessing the information about the replication lag and enabling alerts on excessive lag.
 
 ## Set up replication
 
 You can set up xCluster replication as follows:
 
 1. Open the YugabyteDB Anywhere UI and navigate to **Universes**.
 
-2. Select the universe you want to replicate and navigate to **Replication**.
+1. Select the universe you want to replicate and navigate to **Replication**.
 
-3. Click **Configure Replication** to open the dialog shown in the following illustration:<br>
+1. Click **Configure Replication** to open the dialog shown in the following illustration:<br>
 
-   ![Configure Replication](/images/yp/asynch-replication-2.png)<br>
+   ![Configure Replication](/images/yp/asynch-replication-221.png)
 
-4. Provide the name for your replication.
+1. Provide a name for your replication. This name cannot contain [SPACE '_' '*' '<' '>' '?' '|' '"' NULL].
 
-5. Select the target universe.
+1. Select the target universe.
 
-6. Click **Next: Select Tables**.
+   Note that the paused universes are not included in the selection list.
 
-7. From a list of common tables between source and target universes, select the tables you want to include in the replication and then click **Create Replication**, as per the following illustration:<br>
+1. Click **Next: Select Tables**.
 
-   ![Create Replication](/images/yp/asynch-replication-3.png)
+1. From a list of common tables between source and target universes, select the tables you want to include in the replication, as per the following illustration:
 
-## Configure replication
+   ![Create Replication](/images/yp/asynch-replication-333.png)
 
-You can configure an existing replication as follows:
+   Note that even though index tables are not displayed in the YugabyteDB Anywhere UI, the replication is automatically restarted for them if it is restarted for the main table. If a new index table is created after the replication has been set up, this index table is automatically included in the replication configuration.
```

Comment:
Here, the topic is to create replication but the note talks about restarting the replication. Please change it to the following:

> Note that even though index tables are not displayed in the YugabyteDB Anywhere UI, the replication is automatically set up for them if it is set up for their main table.

--
Also, the following sentence is not correct.
> If a new index table is created after the replication has been set up, this index table is automatically included in the replication configuration.

Please change it to
> If a new index table is created after the replication has been set up, it will not be added to the replication config automatically. To add the new index table to the replication config, you need to restart the replciation for the main table.

Reply:
Changed